contrib/devtools/security-check.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/env python2
```

Comment:
Yes, `security-check` works with python 3 too. The build system calls all these scripts by passing them to the python interpreter, so it doesn't care what is on the first line.